### PR TITLE
feat: create delegate profile @bruno

### PR DESCRIPTION
**Pull request by brunosez**

### CODE REVIEW

**Review by dlecan — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Can you please review your README contribution?
It doesn't respect the expected format.

Here is a raw example:
https://github.com/unik-name/uns-delegates-website/edit/master/docs/delegates/110688c7e2b4ce9f369aade0e5e6d8076a5beb28066d473bcaa4f0ddb124803c/README.md

You should respect YAML and Mardown formats

File: docs/delegates/44908e9ac567d211643bffc82710adf4b0172f13bf4a6bcecc794f156458fc81/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+website: TBD https://example.com
+
+<!--- ## OPTIONAL
+## Display informations about your nodes: uptime, version ...
+#node_ip_addresses:
+--->
+  livenet:
+  
+  - **46.226.107.88  version 5.6.2**, was OVH hosted but fire in Strasbourg's DC drives me to Gandi hosting :-)
```
Comment (dlecan) [inline docs/delegates/44908e9ac567d211643bffc82710adf4b0172f13bf4a6bcecc794f156458fc81/README.md:18]: The version will be automatically extracted by a Bot later

Comment (dlecan) [reply docs/delegates/44908e9ac567d211643bffc82710adf4b0172f13bf4a6bcecc794f156458fc81/README.md:18]: > was OVH hosted but fire in Strasbourg's DC drives me to Gandi hosting :-)

Explain this in the `Summary`, it's a quite funny fact :smile:

File: docs/delegates/44908e9ac567d211643bffc82710adf4b0172f13bf4a6bcecc794f156458fc81/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!---[] OPTIONAL BUT ENCOURAGED
+## social media profiles (twitter, github, forum.uns.network), add your username without @
+--->
+
+github | forum | email | LinkedIn
+------------ | ------------- | -------- | --------
+[brunosez] (http://github.com/brunosez) | Bruno  | brunoconsult@gmail.com | https://www.linkedin.com/in/brunoseznec/
+
+
+website: TBD https://example.com
```
Comment (dlecan) [inline docs/delegates/44908e9ac567d211643bffc82710adf4b0172f13bf4a6bcecc794f156458fc81/README.md:10]: If no website, remove it

**Review by dlecan — CHANGES_REQUESTED** [review]
Much better :clap: 

Do you know to can preview your page?
:point_right: https://deploy-preview-31--uns-delegates.netlify.app/delegates/44908e9ac567d211643bffc82710adf4b0172f13bf4a6bcecc794f156458fc81/

Still to fix:
- website can't be loaded
- several unclickable links

**Comment by brunosez** [comment]
Thanks for the preview site.

I have fixed some blank

is it OK now ?

Le mer. 23 juin 2021 à 17:54, Damien Lecan ***@***.***> a
écrit :

**Review by dlecan — APPROVED** [review]

**Comment by dlecan** [comment]
Great, thank you!
Can you sign your commits, please? 
https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits

**Comment by brunosez** [comment]
Hi Damien ,
i've set up the PGP key, is it ok on your side ?
Rgds , have a nice week :-)

**Comment by dlecan** [comment]
Great, your new commits are signed!
You can sign the previous ones on the branch?

Thank you

**Pull request merged**